metrix-mapping/src/main/java/com/powsybl/metrix/mapping/NetworkTopographyChangeNotifier.java

```diff
@@ -74,6 +74,13 @@ public void afterRemoval(String s) {
         // Do nothing
     }
 
+    @Override
+    public void onUpdate(Identifiable identifiable, String attribute, String variantId, Object oldValue, Object newValue) {
+        if (ATTRIBUTE_BLACK_LIST.contains(attribute)) {
+            this.sendNotification(NotificationType.UPDATE, identifiable);
+        }
```

Comment:
I know this is what was done before, but isn't there a problem here?
The notification is only sent if the attribute is in the **black** list?!

Reply:
it might be a historically badly named variable. I understand it as a list of attributes you want to see the updates on.

Reply:
OK, can you open a new PR to fix this name?